playbooks/playbook.yaml

```diff
@@ -33,9 +33,10 @@
         - V4_CFG_MANAGE_STORAGE
       tags:
         - viya
-    - name: baseline role
+    - name: baseline role install
       include_role:
         name: baseline
+      when: ('baseline' in ansible_run_tags) and ('install' in ansible_run_tags)
       tags:
         - baseline
```

Comment:
I believe this line can be removed as the `when` clause is now taking care of this.

Reply:
I will recheck that to be sure, I thought it wasn't needed at first either but it seems like without a tag, the when clause filtering doesn't even get considered.

Reply:
ok, I can confirm that the `tags:` clause is still required. The when doesn't get considered without it.